src/content/concepts/mode.md

```diff
@@ -80,3 +80,26 @@ module.exports = {
 -  ]
 }
 ```
+
+If you want to change the behavior according the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
```

Comment:
Missing a comma here?

Reply:
Since is my first pull request, I must correct the code and commit again ? 
If yes, what change I have to make ?
Thanks and sorry..

Reply:
@m4ss1m0g Yes, changes should be committed.
Not sure neither, care to explain @dhruvdutt ?

Reply:
Not sure if there should be comma between "*webpack.config.js*" and "you".
Also "according" => "according to"?

Reply:
I'm italian and probably my english is not very good, fell free to correct the punctuation.